### ISSUE TITLE
Change Ui for contact us page 

### ISSUE THREAD

**Issue by hkaur008** [CodeFlow201/codeflow#103]

**Comment by ROHAN842** [comment]
I would like to work on this issue under hactoberfest 2021

**Comment by ROHAN842** [comment]
Hey! I would like to work on this issue under hactoberfest 2021

**Comment by ROHAN842** [comment]
Hey @hkaur008 could you please just see how is this dark theme looking for contact page

![image](https://user-images.githubusercontent.com/61961107/135895123-4bae6a22-5909-4330-9459-8464a8d7816c.png)

**Comment by Aakash-Raj-2001** [comment]
@ROHAN842 hey bro, Nice to meet you here. I want to talk with you bro .Where we can talk together ? Any Socials?

**Comment by ROHAN842** [comment]
You can connect with me on linkedin, my profile is https://www.linkedin.com/in/rohanmodi-79647819a/

**Comment by ROHAN842** [comment]
@hkaur008 shall I move forward with this design mentioned above for contact page....?

**Comment by hkaur008** [comment]
> @hkaur008 shall I move forward with this design mentioned above for contact page....?

Yeah ! Would love it if you can chnage theme according the website theme

**Comment by Janvi-Thakkar** [comment]
@hkaur008 will you please assign it to me?
